packages/aws-cdk-lib/aws-eks/README.md

```diff
@@ -187,7 +187,6 @@ cluster.addNodegroupCapacity('custom-node-group', {
   instanceTypes: [new ec2.InstanceType('m5.large')],
   minSize: 4,
   diskSize: 100,
-  amiType: eks.NodegroupAmiType.AL2_X86_64_GPU,
```

Comment:
Reason for why removing this line
* The section here should focus on `addNodegroupCapacity` but not `amiType`.
* Instance type here `m5.large` does not support `GPU` which is misleading.